src/pages/cli/migration/cli-migrate-aws-account.mdx

```diff
@@ -12,3 +12,10 @@ Run the following steps to migrate an existing project to another AWS account.
 3. Select an AWS Profile that will connect to the new account (Do not use an existing environment)
 
 Amplify CLI updates the `amplify/team-provider-info.json` file with the new environment name. Run `amplify env list` to see an additional environment that is connected to the account of the new profile.
+
+<Callout warning>
+
+If your project includes custom resources, cloning the project to a new environment may not work correctly.
+Ensure that custom resource names are parameterized to guarantee uniqueness when cloning between accounts, regions and Amplify environments
```

Comment:
```suggestion
Ensure that custom resource names are parameterized to guarantee uniqueness when cloning between accounts, regions, and Amplify environments
```

minor nit for Oxford comma since we use it elsewhere 
![image](https://user-images.githubusercontent.com/5033303/215578706-beaccca0-132a-49de-9317-e545e234b95b.png)